receiver/opencensusreceiver/testdata/config.yaml

```diff
@@ -28,17 +25,6 @@ receivers:
       enforcement-policy:
         min-time: 10s
         permit-without-stream: true
-  # The following demonstrates how to disable back pressure, set maximum limits on stream, message size and connection
-  # idle time.
-  # Note: The test yaml has demonstrated configuration on a grouped by their structure; however, all of the settings can
-  # be mix and matched like adding the maximum connection idle setting in this example.
-  opencensus/nobackpressure:
```

Comment:
This configuration example needs to stay because of max-recv-msg-size-mib and max-concurrent-streams. Feel free to name the receiver something else. I can also update it in my OC Receiver documentation.